tests/test_utils.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import pytest
+from lxml import etree
+
+from search.utils import get_content, strip_slashes
+
+
+@pytest.mark.parametrize('input, expected', [
+    ('http://api-gateway/', 'http://api-gateway'),
+    ('/test-path', 'test-path'),
+    ('test-path/', 'test-path'),
+    ('/test-path/', 'test-path'),
+    ('test-path', 'test-path'),
+    ('/test-path/somewhere/', 'test-path/somewhere')
+])
+def test_strip_slashes(input, expected):
+    assert strip_slashes(input) == expected
+
+
+@pytest.mark.parametrize('path, id, file', [
+    ('/scholarly-articles/items', None, 'scholarly-articles-items-response.xml'),
+    ('/scholarly-articles/items', 'article1', 'scholarly-articles-article1-response.xml'),
+])
+def test_get_content(app, requests_mock, path, id, file):
+    gateway = 'http://api-gateway'
+    with app.app_context():
+        asset_path = os.path.join(os.path.dirname(__file__), 'assets', file)
+        xml = etree.parse(asset_path)
+        url = f'{gateway}{path}'
+        if id:
+            url += f'/{id}'
+        requests_mock.get(url, text=etree.tostring(xml, encoding=str))
+
+        app.config['GATEWAY_URL'] = gateway
+        app.config['PATHS_TO_INDEX'] = [path]
```

Comment:
I don't think `PATHS_TO_INDEX` is used in `get_content` so it doesn't need patching for the test? But see comments on the `utils` module itself as they would remove the need to use `app` at all for this test.

Reply:
good point. I think this was left over from a previous implementation. Will remove.